includes/core-changes/vbnewline-is-obsolete.md

```diff
@@ -22,13 +22,5 @@ The [Obsolete](xref:System.ObsoleteAttribute) attribute message for `vbNewLine`
 Visual Basic
 
 #### Affected APIs
-- <xref:Microsoft.VisualBasic.Constants.vbNewLine?displayProperty=fullName>
-
-<!--
```

Comment:
I remember @rpetrusha telling me this is needed for some tooling on the breaking change PR. Please restore.

https://github.com/dotnet/docs/pull/14001#discussion_r322988531

Reply:
Added the comment section back